src/PhpSpec/Console/ResultConverter.php

```diff
@@ -33,6 +50,12 @@ public function convert($result)
             case ExampleEvent::PASSED:
             case ExampleEvent::SKIPPED:
                 return 0;
+
+            case ExampleEvent::PENDING:
+                if ($this->io->isIgnorePendingEnabled()) {
+                    return 0;
+                }
+                break;
         }
 
         return 1;
```

Comment:
While you're modifying this, it makes sense to me for this do the the `default` of the switch